index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+    <head> 
+        <meta charset="utf-8">
+        <title>Responsive Page</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+        <link href="https://fonts.googleapis.com/css?family=Nanum+Gothic" rel="stylesheet">
+    </head>
+    <body>
+        <nav>
+            <span>Navigation</span>
+            <div class="menu_open">
+                <i class="fas fa-bars"></i>
+                <ul>
+                    <li>
+                        <a href="#">PSD WEBSITES</a>
```

Comment:
It's okay, but could you please expand a tag clickable area to make it possible to click on li element?